### ISSUE TITLE
Cursor becomes blurry when hovering the titlebar

### ISSUE THREAD

**Issue by MarcCoquand** [gnunn1/tilix#1605]
On high res display with 200% scaling in wayland the cursor changes to a blurry (low res) cursor when hovering the title bar. I would show a screenshot but the screenshot tool hides the cursor.

I'm running Gnome 3.30, Fedora 29. Display is 2560x1050

**Comment by gnunn1** [comment]
I was able to reproduce it once but only when I first changed the scaling on my second monitor from 100% to 200%. I got the blurry cursor on tilix but has I dragged additional windows onto my second monitor the cursor reverted to a sharp cursor as expected and I haven't been able to reproduce it again. I reset the scaling on my second monitor, power cycled my system and then set the scaling to 200% and it worked fine.

I suspect this is a bug in GTK/Wayland rather then tilix. While tilix does use a custom header bar I'm not doing anything that would cause this as far as I can tell. The blurry cursor is likely a result of upscaling the cursor from 100% to 200% at the GTK/Wayland level and thus not something that Tilix controls.
